### PR TITLE
bump modulesync version from 0.16.10 -> 0.16.11

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

**Comment by dhoppe** [comment]
The file `.travis.yml` has duplicate entries.

**Comment by bastelfreak** [comment]
ah what the hell, why does it fail again...

**Comment by dhoppe** [comment]
Line 4 at `.travis.yml` needs to be removed?

**Comment by bastelfreak** [comment]
@dhoppe this looks really strange. I can't reproduce the errors locally.

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
 sudo: false
+cache: bundler
 before_install:
-  - gem install bundler
   - gem update --system
+  - gem install bundler
 dist: trusty
 rvm:
-  - 2.4.0
+  - 2.3.0
```
Comment (bastelfreak) [inline .travis.yml]: the json gem is in version 2.X in the ruby2.4 version, but one of our dependencies needs 1.X :(
pinning json didn't work out, so I had to downgrade our ruby version

**Pull request merged**